<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWalletDialog</class>
 <widget class="QDialog" name="CreateWalletDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Wallet</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>You are one step away from creating your new wallet!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_subdescription">
     <property name="text">
      <string>Please provide a name and, if desired, enable any advanced options</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="wallet_name_label">
       <property name="text">
        <string>Wallet Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wallet_name_line_edit">
       <property name="minimumSize">
        <size>
         <width>262</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Wallet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="encrypt_wallet_checkbox">
     <property name="toolTip">
      <string>Encrypt the wallet. The wallet will be encrypted with a passphrase of your choice.</string>
     </property>
     <property name="text">
      <string>Encrypt Wallet</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Advanced Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_groupbox">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="disable_privkeys_checkbox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Disable private keys for this wallet. Wallets with private keys disabled will have no private keys and cannot have an HD seed or imported private keys. This is ideal for watch-only wallets.</string>
        </property>
        <property name="text">
         <string>Disable Private Keys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="blank_wallet_checkbox">
        <property name="toolTip">
         <string>Make a blank wallet. Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.</string>
        </property>
        <property name="text">
         <string>Make Blank Wallet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="descriptor_checkbox">
        <property name="toolTip">
         <string>Use descriptors for scriptPubKey management</string>
        </property>
        <property name="text">
         <string>Descriptor Wallet</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="external_signer_checkbox">
        <property name="toolTip">
         <string>Use an external signing device such as a hardware wallet. Configure the external signer script in wallet preferences first.</string>
        </property>
        <property name="text">
         <string>External signer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>wallet_name_line_edit</tabstop>
  <tabstop>encrypt_wallet_checkbox</tabstop>
  <tabstop>disable_privkeys_checkbox</tabstop>
  <tabstop>blank_wallet_checkbox</tabstop>
  <tabstop>descriptor_checkbox</tabstop>
  <tabstop>external_signer_checkbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
